docs/reference/plugins.rst

```diff
@@ -368,7 +368,7 @@ CMSPluginBase Attributes and Methods Reference
         plugin.
 
         Since djangocms-text-ckeditor introduced inline previews of plugins, the icon
-        will not be rendered anymore.
+        will not be rendered any more.
```

Comment:
```suggestion
        will not be rendered anymore.
```

This was the correct use of the word.

https://dictionary.cambridge.org/dictionary/english/anymore

Reply:
Thank you for checking this, Mark. My logic for changing "anymore" to "any more" is because it was being flagged as a spelling error when running `make spelling`. Then I saw that "anymore" as an adverb is used in the US, whereas "any more" as an adverb as well, is used in the UK. The spelling guidelines from Django-CMS docs suggest complying with British English. Please let me know if I should leave it like that or revert the changes. Thank you so much.

Reply:
@luisnavarrodelangel You're right - sorry. I've got this mixed up with the US usage because I didn't read the detail on the Cambridge definition.